Library/Homebrew/utils/formatter.rb

```diff
@@ -31,6 +31,10 @@ def error(string, label: nil)
     label(label, string, :red)
   end
 
+  def wrap(s, width = 172, indentation = 0)
+    " "*indentation + s.gsub(/(.{1,#{width}})(\s+|\Z)/, "\\1\n").gsub("\n", "\n"+" "*indentation) + "\n"
```

Comment:
This is hard to follow. Could you use intermediate variables and/or string interpolation and put each `.gsub` on a new line?